docs/docs/dynamo-store/access-strategies.md

```diff
@@ -0,0 +1,73 @@
+# Access Strategies
+
+The access strategy allows you to opt into custom loading or reloading
+algorithms. For almost all categories in almost all most systems, `Unoptimized`
+is a good choice; you can always switch later.
+
+
+# Unoptimized
+
+The first, and likely most used access strategy is the unoptimized one. It'll
+not apply any optimised reading logic. When loading a stream for the first time
+all of its events will be loaded in batches of `batchSize` (default: `500`).
+This turns out to be a sensible default for most aggregates, especially when
+used in conjunction with the built-in caching layer.
+
+# LatestKnownEvent
+
+There is a special case of stream whose events are all a "full-replace". An
+example could be a customer's contact preferences, or an entity view-data
+stream. This access strategy will ensure that you only load at most a single
+event when transacting or querying such a stream.
+
+# Snapshot(isOrigin, toSnapshot)
+
+Imagine a simple counter service. It does not have a defined start and end as
+most processes would and as such we don't know how large it'll grow over time.
+In order to ensure the snappiness of our application we might apply
+snapshotting to ensure that the state of the counter can be reconstructed in 2
```

Comment:
you dont circle back and mention that there's only one roundtrip though?

Reply:
This was copy pasted from the message db docs, and my tired ass didn’t even read it 🙈